Userland/Libraries/LibWeb/Painting/StackingContext.cpp

```diff
@@ -170,10 +171,10 @@ void StackingContext::paint_internal(PaintContext& context) const
 Gfx::FloatMatrix4x4 StackingContext::get_transformation_matrix(CSS::Transformation const& transformation) const
 {
     auto count = transformation.values.size();
-    auto value = [this, transformation](size_t index, CSS::Length& reference) -> float {
+    auto value = [this, transformation](size_t index, Optional<CSS::Length> reference = {}) -> float {
```

Comment:
The point was to not have to pass a Length when trying to resolve an Angle, because it doesn't need it, but you're right that I didn't explain that. I've now moved that change to the previous commit where the Angle branch was added, mentioned it in the commit message, and stopped passing Lengths to it for transform-functions that don't take LengthPercentages.